### ISSUE TITLE
Ensure all definitions are full sentences, not fragments

### ISSUE THREAD

**Issue by mcharg** [ONRR/nrrd#1893]

**Comment by daniel-coleman-1** [comment]
I think some items are served better by having a full sentence definition, and some items are served better by having fragment definitions.  The best example I'm seeing is the E section:
![Screenshot 2022-01-10 093223](https://user-images.githubusercontent.com/83892362/148802562-fd967af1-1cc0-4bcf-8bc5-a6b00c527cf5.png)
EIS and EITI seem to me to be clearly better served with full sentences, while Extractive Industry seems better served with a more brief fragment definition.

**Comment by daniel-coleman-1** [comment]
@Maroyafaied you're right, as per section 2.23 of the Chicago Manual all glossary terms receive a period regardless of whether they are a full sentence or a fragment. I'll open a PR to review grammar and add punctuations to all glossary items.
